src/logion/model/transaction.model.ts

```diff
@@ -74,8 +75,11 @@ export class TransactionAggregateRoot {
     @Column("numeric", {precision: AMOUNT_PRECISION})
     tip?: string;
 
-    @Column("numeric", {precision: AMOUNT_PRECISION})
-    fee?: string;
+    @Column("numeric", {name: "fee", precision: AMOUNT_PRECISION})
+    inclusion_fee?: string;
+
+    @Column("numeric", {name: "storage_fee", precision: AMOUNT_PRECISION, nullable: true})
+    storage_fee?: string;
```

Comment:
Any reason for not sticking to camel-case ?

Reply:
Ooops, no reason except having done too much Rust lately.